cmd/xmidt-agent/config.go

```diff
@@ -307,6 +324,27 @@ var defaultConfig = Config{
 	Pubsub: Pubsub{
 		PublishTimeout: 200 * time.Millisecond,
 	},
+	Logger: sallust.Config{
+		EncoderConfig: sallust.EncoderConfig{
+			TimeKey:        "T",
+			LevelKey:       "L",
+			NameKey:        "N",
+			CallerKey:      "C",
+			FunctionKey:    zapcore.OmitKey,
+			MessageKey:     "M",
+			StacktraceKey:  "S",
+			LineEnding:     zapcore.DefaultLineEnding,
+			EncodeLevel:    "capital",
+			EncodeTime:     "RFC3339Nano",
+			EncodeDuration: "string",
+			EncodeCaller:   "short",
+		},
+		Rotation: &sallust.Rotation{
+			MaxSize:    1 * 1024 * 1024, // 1MB max/file
```

Comment:
```suggestion
			MaxSize:    1  // 1MB max/file
```

looks like MaxSize's unites are already in megabytes https://pkg.go.dev/gopkg.in/natefinch/lumberjack.v2#section-readme
```
        // MaxSize is the maximum size in megabytes of the log file before it gets
	// rotated. It defaults to 100 megabytes.
```

Reply:
Good catch, thank you.